rtps-r52/tests/mailbox.c

```diff
@@ -26,3 +30,51 @@ int test_mbox_lsio_loopback(void)
     test_end("test_mbox_lsio_loopback", rc);
     return rc;
 }
+
+int test_mbox_rtps_hpps(int mbox_in, int mbox_out){
+	enum hpsc_mbox_test_rc rc = HPSC_MBOX_TEST_SUCCESS;
+	struct hpsc_mbox *mbox_hpps = dev_get_mbox(DEV_ID_MBOX_HPPS_RTPS); //DEV_ID_MBOX_HPPS_TRCH
+	//struct hpsc_mbox *mbox_hpps = dev_get_mbox(DEV_ID_MBOX_LSIO);
+	assert(mbox_hpps);
+
+	test_begin("test_mbox_rtps_hpps");	
+
+	const char* name = "TEST_LINK";
+	struct link *test_link = link_mbox_connect(name, mbox_hpps,mbox_in ,mbox_out , 0, 1); //claim a pair of mailboxes as client. one from channel 3 to 4 and one from channel 4 to 3
+	assert(link);
+
+	int bytes; //bytes read 
+
+	char write_buf [HPSC_MSG_SIZE];
+	char read_buf [HPSC_MSG_SIZE];
+	char payload [HPSC_MSG_PAYLOAD_SIZE];
+
+	memset(&payload, 0, HPSC_MSG_PAYLOAD_SIZE);
+	hpsc_msg_ping(write_buf, HPSC_MSG_SIZE, payload, HPSC_MSG_PAYLOAD_SIZE);
```

Comment:
Pedantic: generally better to use `sizeof(...)` for fixed size arrays instead of referecing the hardcoded lengths in multiple places (only in declaration of variable).